imager/imager_images/admin.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib import admin
+from imager_images.models import Photo, Album
+
+
+class PhotoAdmin(admin.ModelAdmin):
+    fieldsets = [
+        ('Info', {'fields': ['user', 'title', 'description']}),
+        ('Upload image', {'fields': ['picture']}),
+        ('Date information', {'fields': ['date_uploaded', 'date_modified', 'date_published']}),
+        ('Published', {'fields': ['published']}),
```

Comment:
nice!  Fieldsets are a great way to improve the organization and appearance of the admin pages.